BRAD/agent.py

```diff
@@ -218,9 +220,11 @@ def __init__(self,
                 restart = None
 
         if restart is None:
-            new_dir_name = dt.now().strftime("%Y-%m-%d_%H-%M-%S")
+            # making this more human readable
+            new_dir_name = start_path if start_path else dt.now().strftime("%B %d, %Y at %I:%M:%S %p")
+            
             new_log_dir = os.path.join(log_dir, new_dir_name)
-            os.makedirs(new_log_dir)
+            os.makedirs(new_log_dir, exist_ok=True)
```

Comment:
I'm not particularly worried about this potential path sanitization issue, but it would be an improvement.

Mostly, could you just add a note into the docstrings for the new `start_path` parameter